ams.tf

```diff
@@ -81,3 +55,73 @@ resource "azurerm_media_transform" "EncodeToMP" {
   }
 }
 
+
+resource "azurerm_monitor_diagnostic_setting" "ams_1" {
+  name                       = azurerm_media_services_account.ams.name
+  target_resource_id         = azurerm_media_services_account.ams.id
+  log_analytics_workspace_id = module.log_analytics_workspace.workspace_id
+
+  log {
+    category = "MediaAccount"
+
+    retention_policy {
+      enabled = true
+      days    = 14
+    }
+  }
+  log {
+    category = "KeyDeliveryRequests"
+    enabled  = true
+  }
+  metric {
+    category = "AllMetrics"
+
+    retention_policy {
+      enabled = true
+      days    = 14
+    }
+  }
+}
+
+resource "azurerm_media_content_key_policy" "ams_default_policy" {
+  name                        = "PolicyWithClearKeyOptionAndJwtTokenRestriction"
+  resource_group_name         = azurerm_resource_group.rg.name
+  media_services_account_name = azurerm_media_services_account.ams.name
+  description                 = "PRE Content Key Policy"
+  policy_option {
+    name                            = "ClearKeyOption"
+    clear_key_configuration_enabled = true
+    token_restriction {
+      token_type                  = "Jwt"
```

Comment:
Just checked in existing policy this token type string seems to be JWT not "Jwt" i don't know whether it matter or not